src/main/java/jenkins/plugins/nodejs/configfiles/Npmrc.java

```diff
@@ -141,9 +140,9 @@ public String toString() {
      * @throws IOException in case of I/O write error
      */
     public void save(File file) throws IOException {
-        try (Writer writer = new FileWriterWithEncoding(file, UTF_8)) {
-            IOUtils.write(toString(), writer);
-        }
+        BufferedWriter writer = Files.newBufferedWriter(file.toPath(), StandardCharsets.UTF_8, StandardOpenOption.TRUNCATE_EXISTING);
+        writer.write(toString());
+        writer.close();
```

Comment:
Why not?
```suggestion
        try(BufferedWriter writer = Files.newBufferedWriter(file.toPath(), StandardCharsets.UTF_8, StandardOpenOption.TRUNCATE_EXISTING)) {
            writer.write(toString());
        }
```

Reply:
I prefer the previous code more simplier and readable than this without trasformation of file and define TRUNCATE_EXISTING as file operation mode.

Reply:
No problem, I only used `Files` just to stay away from external libraries, but since commons is already included I guess it really doesn't matter :)